pkg/util/steps/runner.go

```diff
@@ -38,6 +42,16 @@ func Run(ctx context.Context, log *logrus.Entry, pollInterval time.Duration, ste
 
 		if err != nil {
 			log.Errorf("step %s encountered error: %s", step, err.Error())
+			if strings.Contains(err.Error(), "ThrottlingLimitExceeded") || strings.Contains(err.Error(), "TooManyRequests") {
```

Comment:
Are we certain that these strings, specifically `TooManyRequests`, will _only_ show up due to storage account throttling? I worry that these are too broad, and we might end up capturing other throttling-related errors that output this error code and mischaracterizing them as StorageAccount errors. 

I think the error handling implemented in pkg/util/storage/manager.go is sufficient for the scope of this error handling story, and would recommend dropping the change here.

Reply:
Removed the check from the runner, as you mentioned we need to cover the scope of the story only for the time being.
Could you please let me know your thoughts.